src/libraries/Microsoft.PowerFx.Interpreter/Functions/Mutation/PatchFunction.cs

```diff
@@ -96,6 +98,18 @@ protected static bool CheckArgs(FormulaValue[] args, out FormulaValue faultyArg)
 
             return retFields;
         }
+
+        protected static IEnumerable<NamedValue> FieldDictToFieldEnumerable(IReadOnlyDictionary<string, FormulaValue> fieldsDict)
```

Comment:
>IEnumerable<NamedValue>  [](http://example.com/codeflow?start=25&length=24)

nit: can we just combine this with  FormulaValue.NewRecordFromField and return RecordValue?  All callers seem to do that